var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -78,17 +79,21 @@ def install(self, spec, prefix):
 
         configure(*options)
         make()
+        make("check")
```

Comment:
> But if you guys want, I'm fine with using self.run_tests for this package.

I guess for now it's better to have an option to turn them off, thereby use `run_tests`.